config/settings/sort.q

```diff
@@ -49,3 +49,8 @@ eodwaittime:0D00:00:10.000		// time to wait for async calls to complete at eod
 \d .servers
 CONNECTIONS:`hdb`tickerplant`rdb`gateway	// list of connections to make at start up
 STARTUP:1b					// create connections
+
+\d .proc
+loadprocesscode:1b  // whether to load the process specific code defined at ${KDBCODE}/{process type}
+reloadcode[`wdb]
```

Comment:
I don't think this should be in config - this will I think load code when loading config, this means things will be loaded in a non-standard order which could cause subtle issues in some places.

Why is this needed? Could we instead set the parentproctype of sort/sortworker to be `wdb` (in process.csv), and then the `wdb` process code would be loaded in these processes as well?